jest/private/jest_config_template.mjs

```diff
@@ -122,6 +123,15 @@ if (updateSnapshots) {
   config.snapshotResolver = bazelSnapshotResolverPath;
 }
 
+if (coverageEnabled) {
+  config.collectCoverage = true;
+  config.coverageDirectory = path.dirname(process.env.COVERAGE_OUTPUT_FILE);
+  config.coverageReporters = [
+    "text",
+    ["lcovonly", { file: path.basename(process.env.COVERAGE_OUTPUT_FILE) }],
+  ];
+}
```

Comment:
Should I take what they already have in their jest config into account, or always overwrite?

Reply:
Could probably overwrite to start, wait to see if anyone would want to customize it more?